setup/setup.sh

```diff
@@ -94,14 +104,25 @@ case $cicd_system_id in
     *) echo "Please see the documentation to learn how to extend to CI/CD sources other than Cloud Build, Tekton, GitLab, CircleCI or GitHub."
 esac
 
+case $incident_system_id in
+    1) INCIDENT_SYSTEM="pagerduty"; read -p "Please enter the PagerDuty Signature Verification Token: " PAGERDUTY_SECRET ;;
+    *) echo "Please see the documentation to learn how to extend to incident sources other than PagerDuty."
+esac
+
+if [ $PAGERDUTY_SECRET != "" ]; then
+    echo $PAGERDUTY_SECRET | tr -d '\n' | gcloud secrets create pager_duty_secret \
```

Comment:
I think we want to do this part in Terraform -  @davidstanke

Reply:
[Here's one way](https://github.com/NandosUK/fourkeys/pull/119) we could bring secrets management under Terraform (targeting this PagerDuty support PR). 🙂

Reply:
Looks pretty good! Once you get that merged in, we'll do a final review and merge this one in!